src/components/BorrowerProfile/LoanBookmark.vue

```diff
@@ -49,7 +57,6 @@ export default {
 				}
 			}
 		`,
-		preFetch: true,
 		preFetchVariables({ route }) {
```

Comment:
The `preFetchVariables` block should be clear to remove here too...

Reply:
Ah yup, good catch. Removed and confirmed to be unneeded.